README.md

```diff
@@ -1,2 +1,5 @@
-# codereviewer1
-Review past project
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
I think you can improve the README file. It's better to include more details about your project, you can use the template we show in the last week.

Reply:
- [ ] What do you think to include some links to make your review as an opportunity for the student to learn -->  [README Template](https://github.com/microverseinc/readme-template) 🦾